### PR TITLE
build: emit typescript declarations in watch

### PR DESCRIPTION
**Pull request by maxkrieger**
Oops!

### CODE REVIEW

File: packages/core/tsconfig.json [hunk 1]
```diff
@@ -24,7 +24,6 @@
     "typeRoots": ["src/types", "node_modules/@types"],
     "esModuleInterop": true,
     "isolatedModules": true,
-    "noEmit": true,
```
Comment (wodeni) [inline packages/core/tsconfig.json:27]: Does this affect the existing build scripts?

Comment (maxkrieger) [reply packages/core/tsconfig.json:27]: good question, I did a quick check with rollup and I believe it overrides this flag anyways. But I am indeed not sure why it was there to begin with, perhaps an artifact of CRA.

**Comment by hypotext** [comment]
This does appear to resolve the cryptic circular build dependency crash I reported earlier. Now, when I add the line `import { shapedefs } from "renderer/ShapeDef";` in `EngineUtils`, the system seems to build and run fine (not reporting any errors either).

**Review by hypotext — APPROVED** [review]
WFM

**Pull request merged**